src/screens/time/components/EditPunchInOutDateTimeCardComponent.tsx

```diff
@@ -0,0 +1,275 @@
+/*
+ * This file is part of OrangeHRM
+ *
+ * Copyright (C) 2020 onwards OrangeHRM (https://www.orangehrm.com/)
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+import React from 'react';
+import {StyleSheet, TouchableOpacity, View} from 'react-native';
+import withTheme, {WithTheme} from 'lib/hoc/withTheme';
+import {connect} from 'react-redux';
+import Card from 'components/DefaultCard';
+import CardContent from 'components/DefaultCardContent';
+import Text from 'components/DefaultText';
+import Divider from 'components/DefaultDivider';
+import {fetchPunchStatus} from 'store/time/punch/actions';
+import Icon from 'components/DefaultIcon';
+import {AndroidEvent} from '@react-native-community/datetimepicker/src/index';
+import {formatTime} from 'lib/helpers/attendance';
+import DateTimePicker from '@react-native-community/datetimepicker';
+import FormattedDate from 'components/FormatedDate';
+
+class EditPunchInOutDateTimeCard extends React.Component<
+  EditPunchInOutDateTimeCardProps,
+  EditPunchInOutDateTimeCardState
+> {
+  constructor(props: EditPunchInOutDateTimeCardProps) {
+    super(props);
+    this.state = {
+      show: false,
+      mode: DATE,
+    };
+  }
+
+  onChange = (event: AndroidEvent, selectedDate?: Date) => {
```

Comment:
`AndroidEvent` :thinking:

Reply:
datetime picker has defined the argument as this. So I have to pass those